.travis.yml

```diff
@@ -17,18 +17,20 @@ matrix:
   include:
     - python: 2.7
       env: TOXENV=py27
+    - python: 2.7
```

Comment:
Can you share why you re-ordered these? At the moment there's a list of testing followed by style environments. With this change it would be testing with style environments mixed in.

Reply:
I thought it was more organised when it's ordered by Python version. But I'll order it like it was before, my bad.